README.md

```diff
@@ -23,12 +23,12 @@ Start to drag & drop the shapes you need
 
 ![Start Drag & Drop the needed shapes](readme-img/002.Start_drag_and_drop_new.gif)
 
-Once the needed shapes are in your whiteboard, you can edit them as much as you want (since these are standard Miro shapes):
+Once the needed shapes are into your whiteboard, you can edit them as much as you want (since these are standard Miro shapes):
```

Comment:
"Once you have added the required shapes onto your whiteboard,"

Reply:
8886cb4 ✅